project/plugins.sbt

```diff
@@ -6,3 +6,5 @@ addSbtPlugin("uk.gov.hmrc"        % "sbt-auto-build"     % "3.22.0")
 addSbtPlugin("uk.gov.hmrc"        % "sbt-distributables" % "2.5.0")
 addSbtPlugin("org.playframework"  % "sbt-plugin"         % "3.0.3")
 addSbtPlugin("org.scoverage"      % "sbt-scoverage"      % "2.0.12")
+addSbtPlugin("org.scalameta"     % "sbt-scalafmt"       % "2.4.6")
+addSbtPlugin("org.scalastyle" %% "scalastyle-sbt-plugin" % "1.0.0" exclude("org.scala-lang.modules", "scala-xml_2.12"))
```

Comment:
please add scalafix instead and add the `prePrChecks` command as per https://github.com/hmrc/pillar2/blob/main/build.sbt#L45

Reply:
Done